TypeCobol/Compiler/CupParser/NodeBuilder/ProgramClassBuilder.cs

```diff
@@ -529,7 +530,15 @@ public virtual void StartTypeDefinitionEntry(DataTypeDescriptionEntry typedef)
             var node = new TypeDefinition(typedef);
             Enter(node, null, symbolTable);
 
-            symbolTable.AddType(node);
+            if (_IsInsideGlobalStorageSection)
+            {
+                node.SetFlag(Node.Flag.GlobalStorageSection, true);
```

Comment:
Can you write a comment to explain why we don't insert typedef in SymbolTable here ?
And also add the text "GLOBALSS_NOTYPEDEF" so we can search for the corresponding rule in #805